app/src/main/java/zapsolutions/zap/util/RefConstants.java

```diff
@@ -32,7 +32,7 @@
     // API request timeouts
     public static final int TIMEOUT_SHORT = 5;
     public static final int TIMEOUT_MEDIUM = 10;
-    public static final int TIMEOUT_LONG = 15;
```

Comment:
Since the timeouts seem to be very strict in case of tor nodes, I wonder if we should add some sort of `multiplier` in case the user is connected to a tor node. The idea is that then every request would be able to e.g. take 3 times longer if its a tor node.

Reply:
Good point, I just added it :+1: